dot/parachain/overseer/overseer.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package overseer
+
+import (
+	"fmt"
+	"github.com/ChainSafe/gossamer/internal/log"
+	"sync"
+)
+
+const COMMS_BUFFER_SIZE = 5
```

Comment:
Following our project's style (also golang) you could rewrite it using camelCase `CommsBufferSize`